cluster/os-3.9.0-alpha.4/README.md

```diff
@@ -0,0 +1,49 @@
+# OpenShift 3.9.0-alpha.4 in ephemeral containers
+
+Provides a pre-deployed OpenShift Origin with version 3.9.0-alpha.4 purely in docker
+containers with qemu. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are the pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+```bash
+export PROVIDER=os-3.9.0-alpha.4
+export VAGRANT_NUM_NODES=0 # currently only one node supported
```

Comment:
@cynepco3hahue, this line can be removed now.I twas the right decision from you to hardcode the number of nodes until we support more nodes for openshift.

Reply:
Oh, the common script needs it set to 0.